owlplug-client/src/main/java/com/owlplug/project/services/PluginLookupService.java

```diff
@@ -52,20 +58,50 @@ public DawPluginLookup lookup(DawPlugin projectPlugin) {
     lookup.setDawPlugin(projectPlugin);
     lookup.setResult(LookupResult.MISSING);
 
-    Iterable<Plugin> plugins = pluginService.find(projectPlugin.getName(), projectPlugin.getFormat());
+    String originalName = projectPlugin.getName();
+    // Normalize the plugin name (remove platform suffixes like x64, x32, etc.)
```

Comment:
I would not normalize during the lookup operation but more during the project sync operation. Early normalization allow to have cleaner *transient* data saved in OwlPlug internal database, and simplify later queries.

If possible, this logic should be moved to the `StudioOneProjectExplorer` when `DAWPlugins.name` is assigned and saved.